### DISCUSSION TITLE
Toggle the theme

### DISCUSSION THREAD

**Discussion by akramloginext** [tamagui/tamagui#3017] [General]
### Current Behavior

I didn't found any way to toggle the theme in my react-native app.
A code example will be helpful for me.
Thanks

### Expected Behavior

I want the same as Tamagui has on their WEB.

example:

https://github.com/tamagui/tamagui/assets/145097270/7300beff-b971-4b31-83ba-9f2e0a0f1408

### Tamagui Version

```markdown
1.91.4
```

### Platform (Web, iOS, Android)

```markdown
Android and iOS
```

### Reproduction

```markdown
n/a
```

**Comment by natew** [comment]
We could port the Onestack.dev light/dark guide now back to Tamagui for sure
